presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -1206,6 +1226,45 @@ public void testDecimalRangeFilters()
         }
     }
 
+    @Test
+    public void testDecimalApproximateAggregates()
+    {
+        // Short decimals with nulls.
```

Comment:
Nit : Remove this line. The test has both Short and Long decimals.

Reply:
Done.